src/main/resources/org/jenkinsci/plugins/stashNotifier/StashNotifier/help-prependParentProjectKey.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  Whether to prepend parent project while forming notify key
```

Comment:
I propose to add a comment that the parent project name is not prepended if the user specified an override key as per [this remark above](https://github.com/jenkinsci/stashnotifier-plugin/pull/60/files#r30741270).